frontend/pages/shelter/pet/add.tsx

```diff
@@ -0,0 +1,443 @@
+import { DeleteOutlined, EditOutlined, EyeOutlined } from "@ant-design/icons";
+import {
+	Breadcrumb,
+	Button,
+	DatePicker,
+	Input,
+	Radio,
+	RadioChangeEvent,
+	Select
+} from "antd";
+import { Content } from "antd/lib/layout/layout";
+import Title from "antd/lib/typography/Title";
+import { FurLength, PetData, Sex, Species, Status } from "common/enums";
+import ShelterLayout from "layouts/shelter/ShelterLayout";
+import moment from "moment";
+import { useRouter } from "next/dist/client/router";
+import React, { ReactNode, useEffect, useRef, useState } from "react";
+import styled from "styled-components";
+
+export default function EditPetDetails() {
+	const router = useRouter();
+	const { id } = router.query;
```

Comment:
this is currently unused. accident or future proof?

Reply:
Forget to remove. Paiseh.